providers/elbv1/elbv1svc/elb_classic.go

```diff
@@ -360,3 +352,35 @@ func (svc *ELBClassicService) GetRegisteredInstances(loadBalancerName string) ([
 	logrus.Debugf("GetRegisteredInstances result: %v", instanceIds)
 	return instanceIds, nil
 }
+
+func (svc *ELBClassicService) findLBTags(loadBalancerNames []string) ([]*elb.TagDescription, error) {
+	var dividedNames [][]string
+	chunkSize := 20
+	for i := 0; i < len(loadBalancerNames); i += chunkSize {
+		end := i + chunkSize
+		if end > len(loadBalancerNames) {
+			end = len(loadBalancerNames)
+		}
+		dividedNames = append(dividedNames, loadBalancerNames[i:end])
+	}
+
+	var tagDescriptions []*elb.TagDescription
+
+	for _, names := range dividedNames {
+		awsNames := make([]*string, len(names))
+		for i, n := range names {
+			awsNames[i] = aws.String(n)
+		}
+
+		params := &elb.DescribeTagsInput{
+			LoadBalancerNames: awsNames,
+		}
+		resp, err := svc.elbc.DescribeTags(params)
+		tagDescriptions = append(tagDescriptions, resp.TagDescriptions...)
```

Comment:
Keep the `err` statement following the assignment, otherwise you're risking a NPE.